modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1328,7 +1328,7 @@ TEST_P(Test_ONNX_layers, Split_EltwiseMax)
     testONNXModels("split_max");
 }
 
-TEST_P(Test_ONNX_layers, LSTM_Activations)
```

Comment:
Please add note why the test is disabled. Here and bellow.

Reply:
As for me, layer name should be `LSTM` but switch implementation class depends on engine settings.

Reply:
Is that comment still actual? To understand why working tests has been disabled and no new alternatives added.